tokio/src/util/linked_list.rs

```diff
@@ -234,26 +234,28 @@ impl<L: Link> fmt::Debug for LinkedList<L, L::Target> {
 // in the list.
 pub(crate) struct CountedLinkedList<L: Link, T> {
     list: LinkedList<L, T>,
-    count: usize,
+    added: u64,
+    removed: u64,
 }
```

Comment:
Be aware that this may merge conflict with #6001.

Reply:
Yes, definitely. Should be an easy resolution, changing the `count: AtomicUsize` into `added: AtomicU64, removed: AtomicU64`.